### ISSUE TITLE
Need to provide dart program launch shortcut with options

### ISSUE THREAD

**Issue by lak-proddev** [eclipse/dartboard#113]
I think we can provide a launch shortcut with options. So user can customize himself.
Like other shortcuts in eclipse (there is a shortcut ends with...)

![image](https://user-images.githubusercontent.com/39690425/62186199-f63faf80-b382-11e9-9ea6-7170f3bdc558.png)

![image](https://user-images.githubusercontent.com/39690425/62186204-ff308100-b382-11e9-973f-00402d5fe1fe.png)

![image](https://user-images.githubusercontent.com/39690425/62186207-03f53500-b383-11e9-8348-92119ad04638.png)

**Comment by jonas-jonas** [comment]
I see where you're coming from. It's a nice shortcut _if_ you know what it does. Personally I think it's quite unintuitive.

But from my point of view, we can add this. 

A side question though: Do you know if it's possible to instead show all associated launch configs for a project? We would probably have to add them programmatically.

**Comment by lak-proddev** [comment]
> A side question though: Do you know if it's possible to instead show all associated launch configs for a project? We would probably have to add them programmatically.

Yes, programmatically it's possible. Do you mean something like this?
![image](https://user-images.githubusercontent.com/39690425/62264780-02894280-b43f-11e9-8def-c14af70e9d4e.png)
